### PR TITLE
Create 2022-05-02-NIAID.md

### PR DESCRIPTION
**Pull request by hannahcheren**
Create new article -- NIAID

Article PR Template

* [ ] Linked the preview url `https://chtc.github.io/article-preview/<article-url>`
* [ ] Looked at the link in https://socialsharepreview.com/ to verify socials
* [ ] Requested reviews from the correct parties

https://chtc.github.io/article-preview/2022/05/02/NIAID.html

### CODE REVIEW

**Comment by hannahcheren** [comment]
https://github.com/CHTC/Articles/blob/development/2022-05-02-NIAID.md

File: 2022-05-02-NIAID.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: "NIAID/ACE - OSG collaboration leads to a successful virtual training session"
+
+author: Hannah Cheren
+
+publish_on:
+  - osg
+canonical_url: <This is a absolute url that points to the canonical site>
```
Comment (CannonLock) [inline 2022-05-02-NIAID.md:8]:
```suggestion
canonical_url: "https://osg-htc.org/spotlights/NIAID.html"
```

**Review by CannonLock — CHANGES_REQUESTED** [review]
Just one thing to change

**Review by CannonLock — APPROVED** [review]
LGTM

**Pull request merged**